pkg/qliksense/context_configs.go

```diff
@@ -287,8 +287,12 @@ func (q *Qliksense) DeleteContextConfig(args []string) error {
 		case qliksenseConfig.Spec.CurrentContext:
 			fmt.Fprintln(out, chalk.Red.Color("Error: Cannot delete current context -"), chalk.Bold.TextStyle(qliksenseConfig.Spec.CurrentContext))
 			fmt.Fprintln(out, chalk.Yellow.Color("Please switch contexts to be able to delete this context."))
+			err := fmt.Errorf("")
```

Comment:
This is a bit confusing to me.. So, we're telling upstream that we have an error here, but not telling them what it is..

Reply:
I added the error invocation specifically so the test cases can catch any error. I will try and put the actual error in that as well but I had specific formatting (color coding) that I dont think Cobra's error handling will be able to print out correctly.

Reply:
fixed it :) should format correctly